sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneEventHubClient.cs

```diff
@@ -155,12 +166,34 @@ internal sealed class TrackOneEventHubClient : TransportEventHubClient
 
             // Build and configure the client.
 
-            var client = TrackOne.EventHubClient.Create(endpointBuilder.Uri, eventHubPath, tokenProvider, clientOptions.DefaultTimeout, transportType);
+            var retryPolicy = (clientOptions.RetryOptions != null)
+                ? new BasicRetryPolicy(clientOptions.RetryOptions)
+                : getDefaultRetryPolicy();
```

Comment:
Didn't think of it, and I probably could... but  I'm not sure that it really buys me anything.  The factory is really just in place to capture the current value of the field, since it could be set by developers with a custom policy.  If that happens, it clears out the options so that they don't propagate down to producers/consumers that get created and the custom policy is used as a default instead.  Am I overlooking anything that would offer a benefit?